README.md

```diff
@@ -1,40 +1,48 @@
-Moonstone [![NPM version](https://img.shields.io/npm/v/moonstone-wrapper.svg?style=flat-square&color=informational)](https://npmjs.com/package/moonstone-wrapper)
-====
+<!-- Made By Isaiah-Hamilton(https://github.com/Isaiah-Hamilton) -->
+
+# Moonstone [![NPM version](https://img.shields.io/npm/v/moonstone-wrapper.svg?style=flat-square&color=informational)](https://npmjs.com/package/moonstone-wrapper)
 
 A simple, standalone, [dogehouse.tv](https://dogehouse.tv/) API wrapper
 
 <!-- GETTING STARTED -->
+
 ## Installing
 
 You will need NodeJS and NPM. To install moonstone on your local project, run:
+
 ```
 npm install --no-optional moonstone-wrapper
```

Comment:
```suggestion
npm install --no-optional --production moonstone-wrapper
```

Add the `--production` flag so you can ignore dev-dependencies. They aren't needed anyways to run the package.